internal/pipe/sign/testdata/README.md

```diff
@@ -8,73 +8,84 @@ key works.
 This uses the `--homedir .` option to create the test keys so that we do not touch
 the local keyring file.
 
-1.  Create signing keys
+## 1.  Create signing keys
 
-    cd $GOPATH/src/github.com/goreleaser/goreleaser/internal/pipeline/sign/testdata/gnupg
-    gpg --homedir . --quick-generate-key --batch --passphrase '' nopass default default 10y
+```sh
+cd $GOPATH/src/github.com/goreleaser/goreleaser/internal/pipeline/sign/testdata/gnupg
+gpg --homedir . --quick-generate-key --batch --passphrase '' nopass default default 10y
+```
 
-1.  Check that the key exists
+## 2. Check that the key exists
 
-    ## $ gpg --homedir . --list-keys
+```sh
+gpg --homedir . --list-keys
+pub rsa2048 2017-12-13 [SC][expires: 2027-12-11]
+FB6BEDFCECE1761EDD68BF32EF2D274B0EDAAE12
+uid [ultimate] nopass
+sub rsa2048 2017-12-13 [E]
+```
 
-    pub rsa2048 2017-12-13 [SC][expires: 2027-12-11]
-    FB6BEDFCECE1761EDD68BF32EF2D274B0EDAAE12
-    uid [ultimate] nopass
-    sub rsa2048 2017-12-13 [E]
+## 3. Check that signing works
 
-1)  Check that signing works
+### 3.1 create a test file
 
-    # create a test file
+```sh
+echo "bar" > foo
+```
 
-    echo "bar" > foo
+### 3.2. sign and verfiy
 
-    # sign and verfiy
+```sh
+gpg --homedir . --detach-sign foo
+gpg --homedir . --verify foo.sig foo
 
-    gpg --homedir . --detach-sign foo
-    gpg --homedir . --verify foo.sig foo
+gpg: Signature made Wed Dec 13 22:02:49 2017 CET
+gpg: using RSA key FB6BEDFCECE1761EDD68BF32EF2D274B0EDAAE12
+gpg: Good signature from "nopass" [ultimate]
+```
 
-    gpg: Signature made Wed Dec 13 22:02:49 2017 CET
-    gpg: using RSA key FB6BEDFCECE1761EDD68BF32EF2D274B0EDAAE12
-    gpg: Good signature from "nopass" [ultimate]
+### 3.3. cleanup
 
-    # cleanup
+```sh
+rm foo foo.sig
+```
 
-    rm foo foo.sig
+## 4. Make sure you have keyrings for both gpg1 and gpg2
 
-1)  Make sure you have keyrings for both gpg1 and gpg2
+travis-ci.org runs on an old Ubuntu installation which
+has gpg 1.4 installed. We need to provide keyrings that
+have the same keys and users for both formats.
```

Comment:
Probably not related to this PR, but this looks a bit out-dated as GoReleaser no longer uses Travis from what I can tell?

Reply:
yeah, still useful to test with both gpg1 and gpg2 I think... 🤔

just fixed formatting, in another pr I'll take a proper look at it...